EnglishTranslation-master/ITEM.tsv

```diff
@@ -10193,46 +10193,46 @@ ITEM_20160224_010192	Rune Stone of Justice
 ITEM_20160224_010193	Rune Stone of Protection
 ITEM_20160224_010194	Rune Stone of Destruction
 ITEM_20160224_010195	Rune Stone of the First Master
-ITEM_20160224_010196	A rune stone made by the first master of the Rune Casters.
+ITEM_20160224_010196	A rune stone made by the first Rune Caster Master.
```

Comment:
The description belongs to "Rune Stone of the First Master", keeping the current description for this item would make it consistent.